README.rst

```diff
@@ -1113,8 +1113,8 @@ fixed but without actually modifying them:
     $ php php-cs-fixer.phar fix /path/to/code --dry-run
 
 Instead of using command line options to customize the rule, you can save the
-project configuration in a ``.php_cs.dist`` file in the root directory
-of your project. The file must return an instance of ``PhpCsFixer\ConfigInterface``,
+project configuration in a ``.php_cs.dist`` file in the root directory of your project.
+The file must return an instance of `PhpCsFixer\\ConfigInterface <https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.2/src/ConfigInterface.php>`_
```

Comment:
Link should be probably general, not locker to the branch, so it up to date with master branch and dead link in case of change is obvious.

`/src/ConfigInterface.php` should do the trick.